### DISCUSSION TITLE
How can I switch from one pull shark achievement repository to another ?

### DISCUSSION THREAD

**Discussion by akhil-ge0rge** [community/community#110882] [Other Feature Feedback, Questions, & Ideas]
I finished my task to pull the shark on. I want to move the existing achievement repository to another one out of a few repositories.

**Comment by thilinaabey** [comment]
To move your existing achievement repository to another repository on GitHub, follow these steps:

1. **Create a Backup:** Before proceeding, ensure you have a backup of your existing repository to prevent data loss.

2. **Navigate to Repository Settings:** Go to the settings tab of your achievement repository on GitHub.

3. **Transfer Repository:** Look for the "Transfer" option in the settings menu. Click on it.

4. **Select Destination Owner:** Enter the username or organization name of the account where you want to move the repository.

5. **Choose Destination Repository:** Select the destination repository from the list of repositories under the chosen account.

6. **Confirm Transfer:** Confirm the transfer by following the prompts. You may need to enter your GitHub password to authenticate.

7. **Verify Transfer:** Once the transfer is complete, verify that the repository has been successfully moved to the new account and repository.

8. **Update Local Repository:** If you have a local copy of the repository, update the remote URL to point to the new repository location using the following Git command:

```bash
git remote set-url origin <new-repository-url>
```

Replace `<new-repository-url>` with the URL of the new repository.

9. **Update Remote References:** If there are any other repositories or services referencing the old repository, update those references to point to the new repository.

10. **Communicate Changes:** Inform relevant stakeholders or collaborators about the repository transfer, especially if they need to update any links or workflows.

By following these steps, you can successfully move your existing achievement repository to another one on GitHub.`

**Comment by Hazrat-Ali9** [comment]
I finished my task to pull the shark on. I want to move the existing achievement repository to another one out of a few repositories.
